### PR TITLE
Prevents decompiler crashing when wrapping catchtype with L and ;

### PR DESCRIPTION
**Pull request by ItzSomebody**
Full example from [here](https://github.com/ItzSomebody/Radon/pull/60).

### CODE REVIEW

**Comment by Storyyeller** [comment]
Is this actually valid Java? Based on the discussion in the linked ticket, it sounds like Java 11 no longer supports this?

**Comment by ItzSomebody** [comment]
[sample.zip](https://github.com/Storyyeller/Krakatau/files/2948043/sample.zip)

No, it is not. Java 11 will spit a LinkageError. The latest Java version I know of which will actually accept this kind of bytecode is Java 8.

File: Krakatau/ssa/blockmaker.py [hunk 1]
```diff
@@ -114,6 +114,8 @@ def __init__(self, parent, iNodes, inputTypes, returnTypes, except_raw, opts):
         self.exceptionhandlers = []
         for (start, end, handler, index) in exceptions:
             catchtype = parent.getConstPoolArgs(index)[0] if index else 'java/lang/Throwable'
+            if catchtype.startswith('L') and catchtype.endswith(';'):
+                catchtype = catchtype[1:len(catchtype) - 1]
```
Comment (Storyyeller) [inline Krakatau/ssa/blockmaker.py]: `len(catchtype) - 1` isn't necessary here. You  can just do [1:-1]

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,4 @@
 test*.py
 Krakatau/plugins/*
 tests/.cache
+.idea/
```
Comment (Storyyeller) [inline .gitignore]: please include a trailing newline

**Comment by Storyyeller** [comment]
Please squash the extra commits to keep the history clean.

**Pull request merged**